package.json

```diff
@@ -1,12 +1,13 @@
 {
   "name": "dotcom",
   "dependencies": {
-    "artillery": "2.0.3",
-    "axios": "1.6.7",
+    "artillery": "2.0.8",
+    "axios": "1.6.8",
     "hot-shots": "10.0.0",
     "node-cron": "3.0.3",
-    "node-json-logger": "0.2.0",
+    "node-json-logger": "0.2.1",
```

Comment:
📓 I just upgraded these while I was already in the file.

Reply:
These updates in the future should be done outside of unrelated changes.